src/Sulu/Bundle/AdminBundle/Resources/js/containers/ViewRenderer/ViewRenderer.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import React from 'react';
+import {getView} from '../../services/ViewRegistry';
+
+export default class ViewRenderer extends React.Component {
+    props: {
+        name: string,
+        parameters?: Object,
+    };
+
+    render() {
+        return React.createElement(getView(this.props.name), this.props.parameters);
```

Comment:
@cpojer The `getView` call in this line is what I try to mock. However, if I do `console.log(getView.toString())` I still see the original implementation...